localstack/services/lambda_/resource_providers/aws_lambda_url.py

```diff
@@ -0,0 +1,132 @@
+# LocalStack Resource Provider Scaffolding v2
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Optional, TypedDict
+
+import localstack.services.cloudformation.provider_utils as util
+from localstack.services.cloudformation.resource_provider import (
+    OperationStatus,
+    ProgressEvent,
+    ResourceProvider,
+    ResourceRequest,
+)
+
+
+class LambdaUrlProperties(TypedDict):
+    AuthType: Optional[str]
+    TargetFunctionArn: Optional[str]
+    Cors: Optional[Cors]
+    FunctionArn: Optional[str]
+    FunctionUrl: Optional[str]
+    InvokeMode: Optional[str]
```

Comment:
This attribute is not being used

Reply:
This part was auto-generated by scaffolding. I would prefer to leave it as it is.